### PR TITLE
Fix LiquidOutputAgent event_limit validations

### PR DESCRIPTION
**Pull request by monsha**
This PR fix the bug mentioned in https://github.com/huginn/huginn/issues/2851

### CODE REVIEW

File: app/models/agents/liquid_output_agent.rb [hunk 1]
```diff
@@ -129,8 +129,8 @@ def validate_options
       end
 
       if options['event_limit'].present?
-        if((Integer(options['event_limit']) rescue false) == false)
-          errors.add(:base, "Event limit must be an integer that is less than 1001.")
+	if (Integer(options['event_limit']) rescue false) == false && date_limit.blank?
```
Comment (dsander) [inline app/models/agents/liquid_output_agent.rb:132]: Can you fix the indentation here? It looks like the two lines shifted to the left.

File: spec/models/agents/liquid_output_agent_spec.rb [hunk 2]
```diff
@@ -72,7 +72,14 @@
       expect(agent).not_to be_valid
     end
 
-    it "should should not allow non-integer event limits" do
+    it "should validate the event_limit with relative time" do
+      agent.options[:event_limit] = "15 minutes"
+      expect(agent).to be_valid
+      agent.options[:event_limit] = "1 century"
```
Comment (dsander) [inline spec/models/agents/liquid_output_agent_spec.rb:78]: 😄 I like that one

**Review by dsander — CHANGES_REQUESTED** [review]
Hi, thanks for fixing the bug!

File: spec/models/agents/liquid_output_agent_spec.rb (hunk 2 above)
Comment (monsha) [reply spec/models/agents/liquid_output_agent_spec.rb:78]: xD

**Review by dsander — APPROVED** [review]

**Review by knu — APPROVED** [review]

**Pull request merged**